tests/jax/components/training/losses_test.py

```diff
@@ -0,0 +1,190 @@
+from types import SimpleNamespace
+from typing import Callable, Dict, Tuple
+
+import jax
+import jax.numpy as jnp
+import pytest
+
+from mava.components.jax.training.losses import (
+    MAPGTrustRegionClippingLossConfig,
+    MAPGWithTrustRegionClippingLoss,
+)
+from mava.systems.jax.trainer import Trainer
+
+
+class MockNet:
+    """Creates a mock network for loss function"""
+
+    @staticmethod
+    def apply(
+        parameters: Dict[str, jnp.ndarray], observation: jnp.ndarray
+    ) -> Tuple[jnp.ndarray, jnp.ndarray]:
+        """Mock function to apply the network to training data"""
+        return observation, parameters["mlp/~/linear_0"]["w"]
+
+
+def log_prob(distribution_params: jnp.ndarray, actions: jnp.ndarray) -> jnp.ndarray:
+    """Mock function to return fixed log probs"""
+    return distribution_params + actions
+
+
+def entropy(distribution_params: jnp.ndarray) -> jnp.ndarray:
+    """Mock function to return fixed entropy"""
+
+    return distribution_params
+
+
+@pytest.fixture
+def mock_trainer() -> Trainer:
+    """Creates mock trainer fixture"""
+
+    observation = SimpleNamespace(observation=jnp.array([0.5, 0.5, 0.7, 0.2]))
+    observations = {
+        "agent_0": observation,
+        "agent_1": observation,
+        "agent_2": observation,
+    }
+
+    agent_net_keys = {
+        "agent_0": "network_agent_0",
+        "agent_1": "network_agent_1",
+        "agent_2": "network_agent_2",
```

Comment:
Please rename the 'agent_net_keys' values here to all be `network_agent` to reflect the shared network defined below.

Reply:
Updated